metrics-facade-base/src/main/java/com/ringcentral/platform/metrics/AbstractMetricRegistry.java

```diff
@@ -242,7 +242,7 @@ protected AbstractMetricRegistry(
     }
 
     public static ScheduledExecutorService makeDefaultExecutor() {
-        return newSingleThreadScheduledExecutor(new BasicThreadFactory.Builder().daemon(true).build());
+        return newSingleThreadScheduledExecutor(new BasicThreadFactory("metric-registry-executor-%d"));
```

Comment:
I think it would be better to specify daemon explicitly.

Reply:
Fixed, thanks